includes/admin.inc

```diff
@@ -602,11 +602,12 @@ function islandora_solr_admin_settings($form, &$form_state) {
   );
   $form['islandora_solr_tabs']['query_defaults']['islandora_solr_base_sort'] = array(
     '#type' => 'textfield',
-    '#title' => t('Sort field for default query'),
+    '#title' => t('Sort field for search results'),
```

Comment:
Not a change request, just a reflexion. Wondering if "Default Sort field(s) for search results" could be more explicit?   We want to say that this sorts will be applied by "default" by any query, but will be overwritten by the sort box if present.

Reply:
We could add "This will be overridden by the Sort block if present" to the help text, but the help text is already pretty long...

The Sort block by its very nature intentionally overrides the default search results sort, so I don't think we need to warn people about it here. Default should always be assumed, except in specific cases where something overrides it. If this fact is not clear in the pages where the Sort block is configured, it's probably better to clarify that there, no?

D&D rules come to mind in this circumstance... Specific overrides general...

Reply:
>We could add "This will be overridden by the Sort block if present" to the help text, but the help text is already pretty long...

Ok, makes sense,w hatever you feel is better. Even maybe documentation (which is only read by folks deploying modules, says my experience) of whatsoever. D & D, good call.  Also D &D provides a  better UX in all the cases compared to this Xmas tree of overlapping fields and toggles. 
What about the extra (s)  in the '#title'? I'm advocation for myself here, I have to support so many UI inconsistency every day with folks asking about that, that I was trying to avoid new tickets. Thanks for you work on this